### PR TITLE
community metadata extension migration

### LINKED ISSUE #340: [CI] should build with try-runtime and runtime-benchmark features as well
currently not tested  (and not working)

### PR DESCRIPTION
**Pull request by brenzi**
integrates https://github.com/encointer/pallets/pull/336

merge this first: https://github.com/encointer/encointer-node/pull/342

### CODE REVIEW

**Comment by brenzi** [comment]
I'm getting an error which predates this PR, but CI didn't catch it until now:

```
  error[E0599]: the function or associated item `try_runtime_upgrade` exists for struct `Executive<Runtime, Block<Header<u32, BlakeTwo256>, UncheckedExtrinsic<MultiAddress<AccountId32, ()>, RuntimeCall, MultiSignature, (CheckSpecVersion<Runtime>, CheckTxVersion<Runtime>, CheckGenesis<Runtime>, CheckMortality<Runtime>, CheckNonce<Runtime>, CheckWeight<Runtime>, ChargeAssetTxPayment<Runtime>)>>, ChainContext<Runtime>, Runtime, (Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>), (Migration<Runtime>, Migration<Runtime>)>`, but its trait bounds were not satisfied
     --> /home/brenzi/encointer/encointer-node/runtime/src/lib.rs:899:28
      |
  899 |             let weight = Executive::try_runtime_upgrade(checks).unwrap();
      |                                     ^^^^^^^^^^^^^^^^^^^ function or associated item cannot be called due to unsatisfied trait bounds
      |
      = note: the following trait bounds were not satisfied:
              `(frame_system::Pallet<Runtime>, pallet_insecure_randomness_collective_flip::Pallet<Runtime>, pallet_timestamp::Pallet<Runtime>, pallet_sudo::Pallet<Runtime>, pallet_balances::Pallet<Runtime>, pallet_transaction_payment::Pallet<Runtime>, pallet_asset_tx_payment::Pallet<Runtime>, pallet_aura::Pallet<Runtime>, pallet_grandpa::Pallet<Runtime>, pallet_utility::Pallet<Runtime>, pallet_proxy::Pallet<Runtime>, pallet_scheduler::Pallet<Runtime>, pallet_treasury::Pallet<Runtime>, pallet_encointer_scheduler::Pallet<Runtime>, pallet_encointer_ceremonies::Pallet<Runtime>, pallet_encointer_communities::Pallet<Runtime>, pallet_encointer_balances::Pallet<Runtime>, pallet_encointer_bazaar::Pallet<Runtime>, pallet_encointer_reputation_commitments::Pallet<Runtime>, pallet_encointer_faucet::Pallet<Runtime>): hidden_include::traits::TryState<u32>`

  error[E0599]: the function or associated item `try_execute_block` exists for struct `Executive<Runtime, Block<Header<u32, BlakeTwo256>, UncheckedExtrinsic<MultiAddress<AccountId32, ()>, RuntimeCall, MultiSignature, (CheckSpecVersion<Runtime>, CheckTxVersion<Runtime>, CheckGenesis<Runtime>, CheckMortality<Runtime>, CheckNonce<Runtime>, CheckWeight<Runtime>, ChargeAssetTxPayment<Runtime>)>>, ChainContext<Runtime>, Runtime, (Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>, Pallet<Runtime>), (Migration<Runtime>, Migration<Runtime>)>`, but its trait bounds were not satisfied
     --> /home/brenzi/encointer/encointer-node/runtime/src/lib.rs:911:15
      |
  911 |             Executive::try_execute_block(block, state_root_check, signature_check, select).expect("execute-block failed")
      |                        ^^^^^^^^^^^^^^^^^ function or associated item cannot be called due to unsatisfied trait bounds
      |
      = note: the following trait bounds were not satisfied:
              `(frame_system::Pallet<Runtime>, pallet_insecure_randomness_collective_flip::Pallet<Runtime>, pallet_timestamp::Pallet<Runtime>, pallet_sudo::Pallet<Runtime>, pallet_balances::Pallet<Runtime>, pallet_transaction_payment::Pallet<Runtime>, pallet_asset_tx_payment::Pallet<Runtime>, pallet_aura::Pallet<Runtime>, pallet_grandpa::Pallet<Runtime>, pallet_utility::Pallet<Runtime>, pallet_proxy::Pallet<Runtime>, pallet_scheduler::Pallet<Runtime>, pallet_treasury::Pallet<Runtime>, pallet_encointer_scheduler::Pallet<Runtime>, pallet_encointer_ceremonies::Pallet<Runtime>, pallet_encointer_communities::Pallet<Runtime>, pallet_encointer_balances::Pallet<Runtime>, pallet_encointer_bazaar::Pallet<Runtime>, pallet_encointer_reputation_commitments::Pallet<Runtime>, pallet_encointer_faucet::Pallet<Runtime>): hidden_include::traits::TryState<u32>`

  For more information about this error, try `rustc --explain E0599`.
  error: could not compile `encointer-node-notee-runtime` due to 2 previous errors

```

File: runtime/Cargo.toml [hunk 1]
```diff
@@ -7,7 +7,7 @@ name = "encointer-node-notee-runtime"
 repository = "https://github.com/encointer/encointer-node/"
 # minor revision must match node/client
 # patch revision must match runtime spec_version
-version = "1.4.25"
+version = "1.4.26"
```
Comment (clangenb) [inline runtime/Cargo.toml]: I do think that this is at least a minor version bump.

**Review by clangenb — APPROVED** [review]
One remark about the versioning, but you can decide. Otherwise, LGTM.

File: runtime/Cargo.toml (hunk 1 above)
Comment (brenzi) [reply runtime/Cargo.toml]: agreed

**Pull request merged**